### DISCUSSION TITLE
Will there be any support for modded items?

### DISCUSSION THREAD

**Discussion by amyfromearth** [FabianMPunkt/Nautilus3D#8] [Ideas]
I'm working on a modpack and I love your textures! Unfortunately anything I add to the pack just doesn't look as nice as the vanilla items you've done. Would you be interested in adding textures for any modded items? 

Specifically I'm looking at Variant Vanilla Blocks, and Farmer's Delight. <3

**Comment by FabianMPunkt** [comment]
Hey I'm sorry but adding models for specific mods would blow the scope of the project and what i can accomplish by myself. Altough, you can submit bug-reports in case you find bugs or incompatabilities between Nautilus3D and other packs / mods.

**Reply by amyfromearth** [reply]
Totally understandable! You've done a great job with these textures and I absolutely love them. <3
